test/DynamoCoreTests/SchedulerTests.cs

```diff
@@ -1026,7 +1038,6 @@ public void TestTaskQueuePreProcessing00()
 
             var expected = new List<string>
             {
-                "FakeUpdateGraphAsyncTask: 6",
```

Comment:
The first `FakeUpdateGraphAsyncTask` (a derived class of `UpdateGraphAsyncTask`) instance is dropped as expected.